### ISSUE TITLE
Barrier Can't Move Pointer from Win 11 to Win 10 v.2.4.0

### ISSUE THREAD

**Issue by syaifulwachid** [debauchee/barrier#1820]
### What happened?

![image](https://user-images.githubusercontent.com/100280103/198738751-84f47861-a39b-49e1-a4de-27f465cacdc3.png)

this is happend on both OS
> server run in windows 11 
>client run in windows 10

when i try to move the pointer mouse from server screen to the client screen it force to leave from client creen back to the server screen directly intime 
as shown in the log"
[2022-10-29T04:39:23] INFO: leaving screen

### Version

v2.4.0

### If applicable, where did you install Barrier from?

i get the file from here:
https://github.com/debauchee/barrier/releases/tag/v2.4.0
![image](https://user-images.githubusercontent.com/100280103/198739098-a474d466-247f-43e8-bdc7-983cfb4b30ce.png)

### What OSes are you seeing the problem on? (Check all that apply)

Windows

### What OS versions are you using?

this is happend on both OS
for the server run in windows 11 
for the client run in windows 10

### Relevant log output

```shell
[2022-10-29T04:39:23] INFO: switch from "YOGA" to "MKU" at 1365,376
[2022-10-29T04:39:23] INFO: leaving screen
[2022-10-29T04:39:23] INFO: bitmap: 1919x1079 32
[2022-10-29T04:39:23] INFO: convert image from: depth=32 comp=3
[2022-10-29T04:39:23] INFO: bitmap: 1919x1079 32
[2022-10-29T04:39:23] INFO: convert image from: depth=32 comp=3
[2022-10-29T04:39:23] INFO: switch from "MKU" to "YOGA" at 190,543
[2022-10-29T04:39:23] INFO: entering screen
```

### Any other information

for the setting I already set as the right setting in it,like open the port on firewall on both client and server, and i laso tried to close/deactivate the windows defender/antivirus on both Laptop as well, and also try restart the both laptop after applying the setting. but the problem is still happen/persist

**Comment by verglor** [comment]
I have the same problem with Win10 server and Win11 22H2 clients.

**Comment by verglor** [comment]
Reverting server to `2.3.4` helped.

**Comment by syaifulwachid** [comment]
> Reverting server to `2.3.4` helped.

yes, i also decide to back use the previous version also work well

**Comment by rp1231** [comment]
I have the same problem of it not connecting between windows 11 and windows 11.
The servers get connected but it doesn't work.....

I encountered this problem in v2.4.0 as well.
I'll try reverting to v2.3.4

**Comment by rp1231** [comment]
Update:

v2.3.4 worked for me but the drag and drop file transfer doesn't seem to be working for me on windows 11.

**Comment by edgan** [comment]
I had this issue, and 2.3.4 works. One of the things I noticed is Windows 11 mentioned no SSL fingerprint.

**Comment by EricChen1248** [comment]
Same issue here. Also noted that revert the W11 to 2.3.4 and W10 still on 2.4.0 still doesn't work as well. Needed to downgrade both for me (did not try W10 on 2.3.4 and W11 on 2.4.0)

**Comment by SDAdham** [comment]
I have the problem between 2 Win11, the mouse doesn't work at all, and if I left click, it will minimize all windows.

**Comment by chrishas35** [comment]
I am having issues after upgrading to Win11 22H2.  My server though is Mac OS 13.1.  The Win11 client logs show the mouse entering, but then complains the server is dead and immediately reconnects.  I'll try the downgrade to 2.3.4 on the Mac OS server.

**Comment by KoolKeith** [comment]
I have the same problem. Using winget I need to go back as far as 2.3.3
